tests/integration_test/services_test.py

```
@@ -29623,6 +29632,292 @@ def _mock_get_bot_settings(*args, **kwargs):
+        f"/api/events/execute/{EventClass.data_importer}",
+    )
+    responses.add(
+        "POST",
+        event_url,
+        json={"success": True, "message": "Event triggered successfully!"},
+    )
+
+    files = (
+        (
+            "training_files",
+            ("nlu.yml", open("tests/testing_data/parallel_action/data/nlu.yml", "rb")),
```

Ruff (0.8.2)

Use a context manager for opening files

(SIM115)